protocol/0018-RSKM-quant_risk_models.ipynb

```diff
@@ -115,7 +116,7 @@
     "1. [ ] If an async update to risk factors is already running, don't start a new one until the previous one has finished. (<a name=\"0018-RSKM-006\" href=\"#0018-RSKM-006\">0018-RSKM-006</a>)\n",
     "1. [ ] Quant risk suite can compute max move up/down ($\\Delta^-$ and $\\Delta^+$) given current price level and a projection horizon such that the resulting prices are within a specified probability level. (<a name=\"0018-RSKM-007\" href=\"#0018-RSKM-007\">0018-RSKM-007</a>)\n",
     "1. [ ] Quant risk suite can compute probability of trading at a given level. (<a name=\"0018-RSKM-008\" href=\"#0018-RSKM-008\">0018-RSKM-008</a>)\n",
-    "1. [ ] Lognormal risk model has defined ranges of valid parameters and market proposals and market update proposals are checked against these. (<a name=\"0018-RSKM-009\" href=\"#0018-RSKM-009\">0018-RSKM-009</a>)"
+    "1. [ ] Lognormal risk model has defined ranges of valid parameters and market proposals and market update proposals are checked against these. The ranges can be found in core (https://github.com/vegaprotocol/vega/blob/c7ef80cff8d8f778b644d94313b9888ba5ad92c2/commands/proposal_submission.go#L820). (<a name=\"0018-RSKM-009\" href=\"#0018-RSKM-009\">0018-RSKM-009</a>)"
```

Comment:
The link isn't what you'd want to use as it points to a specific version of the file; this one is against develop: https://github.com/vegaprotocol/vega/blob/develop/commands/proposal_submission.go#L820